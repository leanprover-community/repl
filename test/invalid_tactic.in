{"cmd": "theorem my_theorem (x : Nat) : x = x := by sorry", "sorries": "individual"}

{"tactic": "exact my_fake_premise", "proofState": 0}
