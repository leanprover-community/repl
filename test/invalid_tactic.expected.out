{"sorries":
 [{"proofState": 0,
   "pos": {"line": 1, "column": 43},
   "goal": "x : Nat\n‚ä¢ x = x",
   "endPos": {"line": 1, "column": 48}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 18},
   "data": "declaration uses 'sorry'"}],
 "env": 0}

{"proofStatus": "Incomplete: contains sorry",
 "proofState": 1,
 "messages":
 [{"severity": "error",
   "pos": {"line": 0, "column": 0},
   "endPos": {"line": 0, "column": 0},
   "data": "Unknown identifier `my_fake_premise`"}],
 "goals": []}

